pmd-java/src/main/java/net/sourceforge/pmd/lang/java/internal/AnnotationSuppressionUtil.java

```diff
@@ -116,7 +117,8 @@ private static Annotatable getSuppressor(Node node) {
             // also works for ASTResource when Resource uses LocalVariableDeclaration
             || node instanceof ASTLocalVariableDeclaration
             || node instanceof ASTFieldDeclaration
-            || node instanceof ASTFormalParameter) {
+            || node instanceof ASTFormalParameter
+            || node instanceof ASTCatchParameter) {
```

Comment:
Would it make sense to test if just for `node instanceof Annotatable` instead?

Reply:
yeah, I don't see why not…